src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Input/Commands/AbstractAddItemCommandHandlerTests.cs

```diff
@@ -202,9 +202,8 @@ public async Task TryHandleCommand_FolderAsNodes_ReturnsTrueWhenUserClicksCancel
         internal AbstractAddItemCommandHandler CreateInstance(IPhysicalProjectTree projectTree = null, IUnconfiguredProjectVsServices projectVsServices = null, IProjectTreeProvider provider = null, IVsAddProjectItemDlg addItemDialog = null, string capability = null)
         {
             var configuredProject = ConfiguredProjectFactory.Create(IProjectCapabilitiesScopeFactory.Create(new string[] { capability ?? TestAddItemCommand.Capability }));
-            projectTree = projectTree ?? IPhysicalProjectTreeFactory.Create(provider);
-            projectVsServices = projectVsServices ??
-                IUnconfiguredProjectVsServicesFactory.Implement(
+            projectTree ??= IPhysicalProjectTreeFactory.Create(provider);
+            projectVsServices ??=                 IUnconfiguredProjectVsServicesFactory.Implement(
```

Comment:
Spacing looks off here.  Can we file a bug on roslyn to fix this?

Reply:
https://github.com/dotnet/roslyn/issues/35870